tests/test_transform.py

```diff
@@ -1,18 +1,39 @@
-from etl import transform  # noqa: F401
+import numpy as np
+from PIL import Image
```

Comment:
I see you are using the PIL package, but it's not in the pyproject.toml or in the poetry.lock, so I can't run the tests to validate them. Can you add the PIL package there?